netdata.spec.in

```diff
@@ -23,6 +23,11 @@ License:	GPL v3+
 Group:		Applications/System
 Source0:	http://firehol.org/download/netdata/releases/v@PACKAGE_VERSION@/%{name}-@PACKAGE_VERSION@.tar.xz
 URL:		http://my-netdata.io/
+%if 0%{?suse_version}
+BuildRequires:	automake
+BuildRequires:	systemd-rpm-macros
+%{?systemd_requires}
```

Comment:
`%{?systemd_requires} is a macro defined in the systemd-rpm-macros package. See https://build.opensuse.org/package/view_file/openSUSE:Factory/systemd-rpm-macros/macros.systemd?expand=1